tests/unit/amo/server/test_base.js

```diff
@@ -560,6 +560,115 @@ describe(__filename, () => {
       expect(users.currentUserWasLoggedOut).toEqual(true);
       mockUsersApi.verify();
     });
+
+    it('sets a X-Accel-Expires header if request is safe & anonymous and response is sucessful', async () => {
+      const { store, sagaMiddleware } = createStoreAndSagas();
+
+      const response = await testClient({ store, sagaMiddleware }).get(
+        '/en-US/firefox/',
+      );
+      expect(response.headers['X-Accel-Expires'.toLowerCase()]).toEqual('180');
```

Comment:
I'd prefer if we had a constant at the top of this file with the x-accel header name in lower case by default, instead of calling `.toLowerCase()`.